### ISSUE TITLE
Pagination - Custom Page Numbers

### ISSUE THREAD

**Issue by vipinbatchu** [AllenFang/react-bootstrap-table#1837]
I am trying to customize the page numbers that get displayed as part of pagination.

By default we have these numbers,

![image](https://user-images.githubusercontent.com/32419507/34593180-1633aeae-f17e-11e7-9499-8de5937d6a85.png)

I am trying to see if I can change the numbers in the un-ordered list to say "Page 1 of 5" or "Page 1 of 10". I tried intercepting the pageList from the paginationPanel and changing the title. Later I realized that these are props and so cannot be changed...(i'm sorry i even tried doing this :()

![image](https://user-images.githubusercontent.com/32419507/34593318-057bd482-f17f-11e7-8de5-24a1dd82b1b1.png)

Is it possible to change the title later in the game?

**Comment by prajapati-parth** [comment]
Hello @vipinbatchu, 
Please try using `paginationShowsTotal` from https://allenfang.github.io/react-bootstrap-table/example.html#pagination | Section: Custom Pagination

**Comment by vipinbatchu** [comment]
@prajapati-parth I tried that already. That option is just a way to show the total text that displays next to the page numbers. I am trying to change the title of the page numbers itself. 

![image](https://user-images.githubusercontent.com/32419507/34707775-16633cb6-f4c4-11e7-80fa-626972cb0981.png)
The code here gets the props from the BootstrapTable and returns a paginationPanel. All the pagination related properties are included in the props. I am asking if there is a way to change the title of the page numbers. Something like | 1 | **_Page 2 of 4_** | 3 | 4 |

**Comment by prajapati-parth** [comment]
@vipinbatchu Oh! Now I got you. I think this(https://allenfang.github.io/react-bootstrap-table/custom.html#pagination) is what you need.

**Comment by vipinbatchu** [comment]
@prajapati-parth Yup that is where I got stuck. 
![image](https://user-images.githubusercontent.com/32419507/34734132-8ec8fbd6-f51f-11e7-86c2-5f31e414b506.png)

I tried to intercept the props.component.pageList and change the page numbers as below

![image](https://user-images.githubusercontent.com/32419507/34734221-dea87d02-f51f-11e7-9c5b-93a60defc4f2.png)

**Comment by AllenFang** [comment]
@vipinbatchu Good point, actually, I think there's not way to custom the page number, we can custom the sizePerPage, pagination list, next/prev page etc but page button is unable to custom. Actually. I can make it customizable but need time to do it.

@prajapati-parth thanks your help 👍
